src/sentry/tasks/integrations.py

```diff
@@ -123,7 +123,11 @@ def sync_status_outbound(group_id, external_issue_id, **kwargs):
     if not has_issue_sync:
         return
 
-    external_issue = ExternalIssue.objects.get(id=external_issue_id)
+    try:
+        external_issue = ExternalIssue.objects.get(id=external_issue_id)
+    except ExternalIssue.DoesNotExist:
```

Comment:
@markstory could we have added it to this line:
```
 @retry(exclude=(ExternalIssue.DoesNotExist, Integration.DoesNotExist))
```
Since this is what we're doing with integrations.

Reply:
Oh nevermind me. Looks like it's already there. Why does that not fix the issue?

Reply:
`ExternalIssue.DoesNotExist` is already part of the retry exclusions for this task. That is what causes the unhandled exception to show up as a sentry issue. I don't think we can do much about someone deleting an issue link while there is a sync job on the queue though.

Reply:
you can prob remove `ExternalIssue.DoesNotExist` from the retry exceptions now that you're handling it, right?